scripts/src/models/orders.ts

```diff
@@ -8,6 +8,7 @@ export type OrderMeta = {
 	title: string;
 	description: string;
 	call_to_action?: string;
+	content?: string;
```

Comment:
looks like the content always contains the same structure.
maybe it should be a typed value instead of a string?

Reply:
this helps with encapsulation of this data across the APIs. I do agree that we can benefit from defining the data structure as an API in the ecosystem-api repo